_posts/molecules/2018-09-27-forms.html

```diff
@@ -11,11 +11,13 @@
 %form
   .form-group
     %label{for: "exampleInputEmail1"} Email address
-    %input#exampleInputEmail1.form-control{"aria-describedby" => "emailHelp", placeholder: "Enter email", type: "email"}
+    %abbr.text-danger{ title: 'required' } *
+    %input#exampleInputEmail1.form-control{"aria-describedby" => "emailHelp", placeholder: "Enter email", type: "email", required: true}
```

Comment:
Space missing inside the brackets. Same for the line 20.

Reply:
@dmarcoux, it was like that before, shall I add the space in all of them?

Reply:
@dmarcoux, done ^^